### PR TITLE
chore(interactions): add `permissions` property to received member

### PR DESCRIPTION
**Pull request by advaith1**
https://github.com/discord/discord-api-docs/pull/2408

### CODE REVIEW

File: v8/payloads/interactions.ts [hunk 1]
```diff
@@ -140,7 +140,9 @@ export type APIInteractionResponsePong = InteractionResponsePayload<APIInteracti
 
 export type APIInteractionResponseAcknowledge = InteractionResponsePayload<APIInteractionResponseType.Acknowledge>;
 
-export type APIInteractionResponseAcknowledgeWithSource = InteractionResponsePayload<APIInteractionResponseType.AcknowledgeWithSource>;
+export type APIInteractionResponseAcknowledgeWithSource = InteractionResponsePayload<
+	APIInteractionResponseType.AcknowledgeWithSource
+>;
```
Comment (advaith1) [inline v8/payloads/interactions.ts:145]: eslint is failing on this but I didn't change this 🤔 probably was changed by eslint in vscode or something

Comment (vladfrangu) [reply v8/payloads/interactions.ts:145]: Update deps, then run lint :p

File: v8/payloads/interactions.ts [hunk 2]
```diff
@@ -84,7 +84,7 @@ export interface APIInteraction {
 	data?: APIApplicationCommandInteractionData;
 	guild_id: string;
 	channel_id: string;
-	member: APIGuildMember & { user: APIUser };
+	member: APIGuildMember & { permissions: string; user: APIUser };
```
Comment (vladfrangu) [inline v8/payloads/interactions.ts:87]: At this point might as well do `APIGuildMember & Required<Pick<APIGuildMember, 'user' | 'permissions'>>` so we can easily add extras on

**Review by vladfrangu — CHANGES_REQUESTED** [review]

File: v8/payloads/interactions.ts (hunk 2 above)
Comment (advaith1) [reply v8/payloads/interactions.ts:87]: permissions isnt in GuildMember

Comment (vladfrangu) [reply v8/payloads/interactions.ts:87]: ..oh
Well
This is why you don't do reviews at 7:42

**Review by kyranet — APPROVED** [review]

**Review by vladfrangu — APPROVED** [review]

**Pull request merged**